weaver/processes/wps_package.py

```diff
@@ -2228,26 +2228,39 @@ def _update_status_dispatch(_provider, _message, _progress, _status):
         app_hints = list(filter(lambda h: any(h["class"].endswith(t) for t in CWL_REQUIREMENT_APP_TYPES), all_hints))
         if len(app_hints) > 1:
             raise ValueError("Package 'requirements' and/or 'hints' define too many conflicting values: {}, "
-                             "only one permitted amongst {}.".format(app_hints, list(CWL_REQUIREMENT_APP_TYPES)))
+                             "only one permitted amongst {}.".format(list(app_hints), CWL_REQUIREMENT_APP_TYPES))
```

Comment:
`app_hints` is already a list (though still ok to leave convert), but `CWL_REQUIREMENT_APP_TYPES` is not a list and must be forced here